pom.xml

```diff
@@ -76,7 +76,7 @@
     <url>https://github.com/payara/Payara</url>
 
     <properties>
-        <version.arquillian_core>1.5.0.Final</version.arquillian_core>
+        <version.arquillian_core>1.1.13.Final</version.arquillian_core>
```

Comment:
Because of this downgrade, I suggest a separate branch for JDK7 compatible Arquillian containers,